### ISSUE TITLE
no way to attach classes to form elements

### ISSUE THREAD

**Issue by mwotton** [tcocca/dynamic_forms#1]
I'm trying to create a checkbox group. Unfortunately, the checkbox_group helper in pp/helpers/dynamic_forms/check_box_group_helper.rb doesn't provide a way to add a class to the generated label.

line 73:            out << "<li>#{markup}<label for='#{id_for(name, item)}'>#{item}</label></li>\n"

I'm passing a custom block in now, but this doesn't help me much in the general run - I'd have to write different code for each different kind of form. Would it be reasonable to attach a class to all form elements?

**Comment by tcocca** [comment]
This sounds like a good idea.  Maybe we should take this even further and add support for formtastic or something like that?

~ Tom

**Comment by mwotton** [comment]
yes, that'd be good. we ended up doing a lot more of it manually than I'd really have liked, formtastic support would have been excellent.

**Comment by tcocca** [comment]
Hi Mark, any chance you want to take a stab at this feature of adding formtastic support?  Or even opening it up to adding more html option support to the fields somehow?

~ Tom

**Comment by mwotton** [comment]
@tcocca I'd like to, but I'm pretty pressed for time at the moment. will circle back when the current massive pile of work is looking less intimidating.

**Comment by tcocca** [comment]
Sounds like we are both in the exact same predicament.  Hopefully that clears up soon.  Keep me posted on this, don't want to end up doing double work.

~  Tom
